### DISCUSSION TITLE
RPC Error- Subgraph Studio on zkSync Era

### DISCUSSION THREAD

**Discussion by tevanrobin** [zkSync-Community-Hub/zksync-developers#99] [General] [answered]
Hi team, we've deployed a new subgraph using Subgraph Studio on the zkSync Era. However, it has only synced about 75% and then halted for over 2 days without any further progress. 

We submitted this to graph team and they confirmed it as an issue with RPC error and they are in touch with you.

Could you kindly assist us with this issue? 

Here's our deployment ID: QmYRdAj84KTvK9q9WEmzBC4N7L8tQWTiJa6hHxBX9n8TYT.

![image](https://github.com/zkSync-Community-Hub/zkync-developers/assets/122250594/0d207bbe-d45f-41ba-b8db-586690046c73)![image](https://github.com/zkSync-Community-Hub/zkync-developers/assets/122250594/03d66e86-ac1e-4938-b0c3-940ab4cf3e57)

**Comment by uF4No** [comment, marked as answer]
Hi

Yeah we identified an issue with responses being too large and a fix was deployed on August 10th. Please let us know if that's resolved on your end.

Apart from that and just to share the info, [theGraph CLI version 0.55.0 now supports both zkSync Era testnet and mainnet](https://github.com/graphprotocol/graph-tooling/blob/main/packages/cli/CHANGELOG.md), in case you want to use `graph init --studio` to scaffold subgraphs.

**Comment by bricnounlerumark** [comment]
Sure, I'll look into the issue with the subgraph deployment using the provided deployment ID. I'll coordinate with the graph team to resolve the RPC error and ensure that the subgraph syncs properly. Thank you for bringing this to our attention.

**Comment by platactiomeb** [comment]
Deployment ID: QmYRdAj84KTvK9q9WEmzBC4N7L8tQWTiJa6hHxBX9n8TYT

Issue: Subgraph deployment on zkSync Era halted at 75% sync progress due to RPC error.

Resolution: The Graph team is in touch with zkSync regarding the RPC error to address the syncing issue.

**Comment by lupanncoofonpowell** [comment]
Issue: Subgraph deployment on zkSync Era halted at 75% syncing for over 2 days due to RPC error.
Status: Graph team confirmed the issue and is in touch with zkSync team for resolution.
Deployment ID: QmYRdAj84KTvK9q9WEmzBC4N7L8tQWTiJa6hHxBX9n8TYT.

**Comment by epjocnopasara** [comment]
Certainly! We'll investigate the reported issue with your subgraph deployment on zkSync Era and work with the Graph team to resolve the RPC error causing the sync to halt. Thank you for providing the deployment ID. We'll update you on our progress shortly.

**Comment by venlaudecsubs** [comment]
The user has deployed a new subgraph on the zkSync Era using Subgraph Studio, but it has only synced about 75% and then halted for over 2 days without any further progress. The user has submitted this issue to the graph team, who confirmed it as an RPC error and are in touch with the zkSync team. The user is requesting assistance with this issue and has provided the deployment ID: QmYRdAj84KTvK9q9WEmzBC4N7L8tQWTiJa6hHxBX9n8TYT.
